src/Command/CreateEntity/User/BulkUserCreationCommand.php

```diff
@@ -0,0 +1,227 @@
+        }
+    }
+
+    /**
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->symfonyStyle->comment('Executing Bulk user creation');
+
+        try {
+            $processDataResult = $this->bulkCreateUsersService->createUsers(
+                $this->lineItemIds,
+                $this->lineItemSlugs,
+                $this->userPrefix,
+                $this->batchSize,
+                $input->getOption(self::OPTION_GROUP_PREFIX)
+            );
+            if (!empty($processDataResult->getNonExistingLineItems())) {
+                $this->symfonyStyle->note(
+                    sprintf(
+                        'Line Items with slugs/ids \'%s\' were not found in the system.',
+                        implode(',', $processDataResult->getNonExistingLineItems()),
+                    )
+                );
+            }
+            $this->symfonyStyle->success(
+                sprintf('%s', $processDataResult->getMessage())
+            );
+        } catch (Throwable $exception) {
+            $this->symfonyStyle->error($exception->getMessage());
+
+            return self::FAILURE;
+        }
+
+        return self::SUCCESS;
+    }
+
+    private function initializeLineItemIdsOption(string $inputLineItemIds): void
+    {
+        $lineItemIds = array_filter(
+            explode(',', $inputLineItemIds),
+            static function (string $value): bool {
+                return !empty($value) && filter_var($value, FILTER_VALIDATE_INT);
+            }
+        );
+
+        if (empty($lineItemIds)) {
+            throw new InvalidArgumentException(
+                sprintf('Invalid %s option value received.', self::OPTION_LINE_ITEM_IDS)
+            );
+        }
+        $this->lineItemIds = array_map('intval', $lineItemIds);
+    }
+
+    private function initializeLineItemSlugsOption(string $inputLineItemSlugs): void
+    {
+        $this->lineItemSlugs = array_filter(
+            explode(',', $inputLineItemSlugs)
+        );
+
+        if (empty($this->lineItemSlugs)) {
+            throw new InvalidArgumentException(
+                sprintf('Invalid %s option value received.', self::OPTION_LINE_ITEM_SLUGS)
+            );
+        }
+    }
+
+    private function initializeUserPrefixOption(string $inputUserPrefix): void
+    {
+        $this->userPrefix = array_filter(
+            explode(',', $inputUserPrefix)
+        );
+
+        if (empty($this->userPrefix)) {
+            throw new InvalidArgumentException(
+                sprintf('User Prefix is a required argument')
+            );
+        }
+    }
+
+    private function initializeBatchOption(string $inputBatchSize): void
+    {
+        if (filter_var($inputBatchSize, FILTER_VALIDATE_INT)) {
+            $this->batchSize = (int) $inputBatchSize;
+        }
+
+        if (empty($this->batchSize)) {
```

Comment:
```suggestion
        $this->batchSize = filter_var($inputBatchSize, FILTER_VALIDATE_INT)); 

        if ($this->batchSize === false) {
```

Reply:
As per the suggested change , it works fine for valid batch number . But if we enter an invalid integer number like(-b 4e or -b 4.5) for batch will get an error "$batchSize must be int, bool used".

Reply:
As per the suggested change ,if we enter an invalid integer number like(-b 4e or -b 4.5) for batch will get an error "$batchSize must be int, bool used". This is because $batchSize is already an int type variable and for  an invalid batch number  after validating   FILTER_VALIDATE_INT will return bool (false) value